sydent/sms/openmarket.py

```diff
@@ -67,17 +87,25 @@ def sendTextSMS(self, body, dest, source=None):
                 "address": source['text'],
             }
 
-        b64creds = b64encode(b"%s:%s" % (
-            self.sydent.cfg.get('sms', 'username'),
-            self.sydent.cfg.get('sms', 'password'),
-        ))
+        # Make sure username and password are bytes otherwise we can't use them with
+        # b64encode.
+        username = self.sydent.cfg.get('sms', 'username')
+        password = self.sydent.cfg.get('sms', 'password')
+
+        if not isinstance(username, bytes):
+            username = username.encode("UTF-8")
```

Comment:
could these be unconditional? `username.encode` should work fine on a `str` for either py2 or py3.